crates/bevy_audio/src/audio_output.rs

```diff
@@ -65,19 +67,54 @@ where
         })
     }
 
+    fn play_spatial_source(
+        &self,
+        audio_source: &Source,
+        repeat: bool,
+        spatial: SpatialSettings,
+    ) -> Option<SpatialSink> {
+        self.stream_handle.as_ref().map(|stream_handle| {
+            let sink = SpatialSink::try_new(
+                stream_handle,
+                spatial.emitter,
+                spatial.left_ear,
+                spatial.right_ear,
+            )
+            .unwrap();
```

Comment:
I'm not convinced this should unwrap here: why not just return `None`?

Reply:
it's what was done on the not spatial side... I changed both to return `None` and warn on the error